core/platform/search/gae_search_services.py

```diff
@@ -48,27 +48,27 @@ def add_documents_to_index(documents, index, retries=DEFAULT_NUM_RETRIES):
     """Adds a document to an index.
 
     Args:
-      - documents: a list of documents. Each document should be a dictionary.
-          Every key in the document is a field name, and the corresponding
-          value will be the field's value.
-          If there is a key named 'id', its value will be used as the
-          document's id.
-          If there is a key named 'rank', its value will be used as
-          the document's rank.
-          By default, search results are returned ordered by descending rank.
-          If there is a key named 'language_code', its value will be used as
-          the document's language. Otherwise, constants.DEFAULT_LANGUAGE_CODE is
-          used.
-      - index: the name of the index to insert the document into, a string.
-      - retries: the number of times to retry inserting the documents.
+        documents: a list of documents. Each document should be a dictionary.
```

Comment:
Optional: Add typeinfo, if possible (here and elsewhere in this file for Args/Returns).

Reply:
Done